docs/troubleshooting-guide/sql.md

```diff
@@ -0,0 +1,40 @@
+# What type of problem do you have with your SQL question?
+
+## [I'm getting a SQL error message][sql-error-message].
+
+- The error message appears in red text when you run a question that uses the [SQL editor][sql-editor].
+- The error message contains part of your SQL query, such as a column or table name.
+- The error message may also contain a three-digit API error code, such as `400` or `404`.
+
+## My SQL query results are incorrect.
+
+- [My query uses joins](/learn/sql-questions/sql-join-types.html#common-problems-with-sql-joins).
+- [My dates and times are wrong](/docs/latest/troubleshooting-guide/timezones.html).
+
+## My SQL variables aren't working.
+
+What type of [SQL variable][sql-variable-def] are you using?
+
+### My SQL variable type is "field filter".
```

Comment:
I'd like to keep "variable type" in the heading because that's how the feature appears in the product (i.e. the menu lists options under "variable type" instead of "filter type"). 

If the user's mental model depends on the term "filter", they'll be able to get to these resources from the Filters page > SQL question > Troubleshooting SQL variables.